.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 name: Build SwiftLeeds
 
 on:
-  pull_request:
+  # pull_request:
```

Comment:
🚨  This PR is a significant and breaking change to our database structure.

Migrations have been put in place to ensure everything transfers seamlessly. However, if we did this on a branch deploy it would break production.

I have disabled branch deployments until this gets merged in. I will revert this in a subsequent PR.